switch_model/wecc/get_inputs/post_process_steps/add_storage.py

```diff
@@ -101,28 +104,46 @@ def drop_previous_candidate_storage():
 @register_post_process(
     name="add_storage",
```

Comment:
So shouldn't this be removed as well as priority? There are a few post process steps that still have old properties.